specs/daito/Unique Durable Identifier for Volumes.md

```diff
@@ -0,0 +1,147 @@
+﻿# Unique Durable Identifier for Volumes
+
+**Author(s)**: [Najmudheen](https://github.com/NajmudheenCT)
+
+## Summary
+
+This proposal brings out the design details to implement  an additional identifier for the volumes in OpenSDS, which can be used by any  management platforms, applications, plugins  etc  to uniquely identify the Volume which is backing the  virtual disk.
+
+## Motivation
+Host platforms like VMware uses generic  algorithm  to generate a unique id for Virtual Disks. Management modules who manage end to end path (plugins or SRMs) can use this identifier to associate the relation . A common method used by Host Platforms is a  SCSI Inquiry command to the VPD page to retrieve   some "metadata" of  target LUN. This id will  persist between restarts , attaching to a different machine or a different SCSI port of the same machine. There are different standards followed in industry for unique identifier like NAA, EUI FC_WWN etc.
+
+Having this identifier details in volume spec would help these platforms to associate vDisk to Volume in OpenSDS.
+
+
+### Goals
+
+* Define an Identifier structure as used standard storage APIs like  swordfish.
+* Define modified  Volumespec with Identifier.
+* Include  Identifier in  API response.
+
+### Non-Goals
+
+* How driver assign identifier is not covered in this design doc.
+
+## Design Details
+
+### Identifier 
+This type describes any additional identifiers for a volume.
+
```

Comment:
please add swordfish reference link here

Reply:
Added reference to Redfish schema document.
Thanks,